scripts/L1_funs.R

```diff
@@ -31,7 +31,7 @@ createGrid <- function(tileSz){
   gridInfo$Use <- NA
   return(gridInfo)
 }
-standardizePC <- function(gridN, gridInfo, catObj, overlap, dirPath, type){
+standardizePC <- function(gridN, gridInfo, catObj, overlap, dirPath, type, ROI){
```

Comment:
`ROI` should be `ROI=NULL` in the arguments to be default right? Otherwise it would be a missing argument.

I tested this with a sample function but I could be missing something! I am completely biased about how I type code.

Reply:
Not at all, you're right again! I decided this makes most sense as an optional element after I processed the data, thanks for the catch.